completion_aggregator/models.py

```diff
@@ -171,8 +172,44 @@ def submit_completion(self, user, course_key, block_key, aggregation_name, earne
                 'last_modified': last_modified,
             },
         )
+        self.emit_completion_aggregator_logs([obj])
+
         return obj, is_new
 
+    @staticmethod
+    def emit_completion_aggregator_logs(updated_aggregators):
+        """
+        Emit a tracking log for each element of the list parameter.
+
+        Parameters
+        ----------
+            updated_aggregators: List of Aggregator intances
+
+        """
+        for obj in updated_aggregators:
+            event = "progress" if obj.percent < 1 else "completion"
+            event_type = obj.aggregation_name
+
+            if event_type not in settings.ALLOWED_COMPLETION_AGGREGATOR_EVENT_TYPES.get(event, []):
+                continue
+
+            event_name = f"edx.completion_aggregator.{event}.{event_type}"
+
+            tracker.emit(
+                event_name,
+                {
+                    "user_id": obj.user_id,
+                    "course_id": str(obj.course_key),
+                    "block_id": str(obj.block_key),
+                    "modified": obj.modified,
+                    "created": obj.created,
+                    "earned": obj.earned,
+                    "possible": obj.possible,
+                    "percent": obj.percent,
```

Comment:
This doesnt need anything change.
I mean this serializer is ok?
https://github.com/nelc/openedx-completion-aggregator/blob/ead8f61b1081a87aa7e37887d82724f807320129/completion_aggregator/models.py#L250

Reply:
Beetween [0,1]?

Reply:
That method just validates the kind of event (progress or completion) the data that is sent must be as accurate as possible